tests/config/pytest_modules.yml

```diff
@@ -1468,9 +1468,9 @@ nucmer:
 oncocnv:
   - modules/nf-core/oncocnv/**
   - tests/modules/nf-core/oncocnv/**
-openms/idmerger:
-  - modules/nf-core/openms/idmerger/**
-  - tests/modules/nf-core/openms/idmerger/**
+openms/peakpickerhires:
```

Comment:
don't put these back, I've already ported to nf-test

Reply:
those had me confused. I thought I deleted these lines, but then they appeared as a merge conflict.
So just to be clear, those lines are deprecated since they specified the pytests, should not be in the `pytest_modules.yml` anymore and hence no further action needs to be taken here?

Reply:
Yes, openms/peakpickerhires has been ported to nf-test already, so delete these three lines, as you did with openms/idmerger  ;)